content/en/continuous_integration/pipelines/circleci.md

```diff
@@ -52,6 +52,31 @@ To set custom tags to all the pipeline and job spans generated by the integratio
 #### Integrate with Datadog Teams
 To display and filter the teams associated with your pipelines, add `team:<your-team>` as a custom tag. The custom tag name must match your [Datadog Teams][8] team handle exactly.
 
+### Configuring multiple projects in bulk
```

Comment:
```suggestion
### Configure multiple projects in bulk
```
for consistency with other titles

Reply:
I would move the section up right before `### Set custom tags`. I think it makes more sense there